var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -7,57 +7,61 @@
-    version('2.0.0', sha256='4eeedf4404824d8de6e4ef580b8a703c0aedb5c03f900f5ce6f85f0b35980135')
-
+    version(
+        '2.5.0', sha256='7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329')
+    version(
+        '2.4.0', sha256='50ecea04b1e41c88835b4b3fd4e7bf0a0a2a3129855c9cc4ba6cf6a1575106e2')
+    version(
+        '2.3.1', sha256='3bf81ccc20a7f2715935349336a76ba4c8402355e1dc3848fcd6f4c3c5931893')
+
+    # general build options
+    variant('mpi', default=True, description='Enable MPI')
+    variant('build_type', default='Release',
+            description='CMake build type',
+            values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
     variant('shared', default=True,
             description='Also build shared libraries')
     variant('pic', default=True,
             description='Enable position independent code '
                         '(for usage of static in shared downstream deps)')
-    variant('mpi', default=True,
-            description='Enable MPI')
+    variant('endian_reverse', default=False,
+            description='Enable Endian Interoperability')
 
-    # transforms
+    # compression libraries
     variant('blosc', default=True,
-            description='Enable Blosc transforms')
+            description='Enable Blosc compression')
     variant('bzip2', default=True,
-            description='Enable BZip2 transforms')
+            description='Enable BZip2 compression')
     variant('zfp', default=True,
-            description='Enable ZFP transforms')
+            description='Enable ZFP compression')
     variant('png', default=True,
-            description='Enable ZFP transforms')
+            description='Enable PNG compression')
+    variant('sz', default=True,
+            description='Enable SZ compression')
 
-    # sz is broken in 2.2.0: https://github.com/ornladios/ADIOS2/issues/705
-    # variant('sz', default=True,
-    #         description='Enable SZ compression')
     # transport engines
+    variant('sst', default=True,
+            description='Enable the SST staging engine')
     variant('dataman', default=True,
             description='Enable the DataMan engine for WAN transports')
-    # currently required by DataMan, optional in the future
-    # variant('zeromq', default=False,
-    #         description='Enable ZeroMQ for the DataMan engine')
+    variant('ssc', default=True,
+            description='Enable the SSC staging engine')
     variant('hdf5', default=False,
             description='Enable the HDF5 engine')
-    variant('adios1', default=False,
-            description='Enable the ADIOS 1.x engine '
-                        '(in 2.3.0+ integrated in BPFile engine)')
-    # language bindings
-    variant('python', default=False,
-            description='Enable the Python >= 2.7 bindings')
+
+    # optional language bindings, C++11 and C always provided
+    variant('python', default=True,
+            description='Enable the Python >= 3.5 bindings')
```

Comment:
Leave the version out now that you once again support Python 2.

Reply:
Great catch, thanks!